post/views.py

```
@@ -0,0 +1,113 @@
+    ## 그리고 이건 사실상 id 매개변수가 들어있는 post 느낌이겠지. 그냥 get + post라고 생각하면 될듯.
+    @swagger_auto_schema( # 데코레이터는 request 있는 post와 비슷하게.
+            operation_id='게시글 수정',
+            operation_description='게시글을 수정합니다.',
+            request_body=PostSerializer, # request body의 '구조'를 설명하는.
+            responses={200: PostSerializer, 404: 'Not Found', 400: 'Bad Request'}
+        )
+    def put(self, request, post_id):
+        # 먼저 get_id와 비슷하게 하고.
+        try: 
+            post = Post.objects.get(id=post_id)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)